### DISCUSSION TITLE
Add timeout for SOC PCAP jobs.

### DISCUSSION THREAD

**Discussion by samanosuke26** [Security-Onion-Solutions/securityonion#12039] [Ideas]
Incomplete jobs (due to PCAP having already rolled off the sensor) seems to stick around indefinitely. I found a job from 2 months ago that was on it's 9,000th attempt. Having the pcap jobs "fail" after a certain number of attempts would be useful.

**Comment by petiepooo** [comment]
I will add that it would be useful to have all PCAP jobs expire after some time.  Long-running sensors can end up using considerable space storing old PCAP jobs.

**Reply by samanosuke26** [reply]
Seconded!

**Comment by TOoSmOotH** [comment]
I created an issue for this.

https://github.com/Security-Onion-Solutions/securityonion/issues/12054
